assets/js/modules/analytics/components/settings/SettingsView.js

```diff
@@ -95,6 +100,23 @@ export default function SettingsView() {
 						<DisplaySetting value={ profileID } />
 					</p>
 				</div>
+				<div className="googlesitekit-settings-module__meta-item">
+					<h5 className="googlesitekit-settings-module__meta-item-type">
+						&nbsp;
+					</h5>
+					<Link
+						href={ editViewSettingsURL }
+						external
+						className="googlesitekit-settings-module__inline-link"
```

Comment:
This seemed the safest way to override using CSS without making cascading changes through the whole site

Reply:
Change of plan: remove this `className` and instead wrap the links with `<p className="googlesitekit-settings-module__meta-item-data">`

Reply:
See my previous comment about placement. If needed, I think the `inherit` prop of the `Link` component should be all we need to match the surrounding text.

Reply:
Much better. It works

![works](https://user-images.githubusercontent.com/8496063/127862874-3d11f539-3dc5-4d8a-9f15-49cd36c7e761.png)